testing/profile.go

```diff
@@ -60,8 +60,8 @@ type Profile struct {
 type expectedOutput struct {
 	Headers           map[string]string `yaml:"headers,omitempty"`
 	Data              interface{}       `yaml:"data,omitempty"` // Accepts array or string
-	LogContains       string            `yaml:"log_contains,omitempty"`
-	NoLogContains     string            `yaml:"no_log_contains,omitempty"`
+	LogContains       []string          `yaml:"log_contains,omitempty"`
```

Comment:
for compatibility maybe we could use interface{} and when processing the variable we can switch between string and []string